debian/tests/mount

```diff
@@ -0,0 +1,123 @@
+#!/usr/bin/python3
+
+"""Mount test for official models.
+
+The official models are here:
+
+    http://people.canonical.com/~vorlon/official-models/
+
+For each of these, build an actual disk image.  For each image, go through the
+mountable partitions as defined in the gadget.yaml (e.g. not the mbr, but
+including the unspecified last writable partition), and make sure each
+partition can actually be mounted.
+"""
+
+import os
+import sys
+
+from contextlib import ExitStack
+from subprocess import PIPE, run
+from ubuntu_image.parser import FileSystemType, parse
+
+
+TMP = os.environ['AUTOPKGTEST_TMP']
+DIR = os.path.abspath(os.path.join('debian', 'tests', 'models'))
+STATUS = {}
+
+
+for model_file in os.listdir(DIR):
+    model_status = STATUS.setdefault(model_file, {})
+    print('==========>', model_file, flush=True)
+    base, suffix = os.path.splitext(model_file)
+    if suffix != '.assertion':
+        continue
+    # Build the image.
+    workdir = os.path.join(TMP, base)
+    model_assertion = os.path.join(DIR, model_file)
+    run(['ubuntu-image', '-c', 'edge', '-w', workdir, model_assertion])
```

Comment:
Is 'edge' the right channel to use for the tests?  Do we risk a high degree of test flakiness because of breakage in the snaps on this channel?  (AIUI, ubuntu-image -c beta should now work - not sure that it does yet for stable)